javascript/gradio.js

```diff
@@ -5,3 +5,6 @@ Object.defineProperty(Array.prototype, 'toLowerCase', {
         return this;
     }
 });
+
+// disable console.warn
+console.warn = function(){};
```

Comment:
It would be better to fix the warnings rather than hiding them.

Reply:
simply removed. `console.warn` is not related with this issue.